index.html

```diff
@@ -141,8 +141,8 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
 an authority (e.g. DNS Registrars).
           </li>
           <li>
-The controller of a DID can cryptographically authenticate themselves
-(e.g. DID-based website login) .
+DIDs enable the controller of a DID to cryptographically authenticate themselves
+(e.g. DID-based website login).
```

Comment:
We could even put "DID-based website login using a WebAuthn/FIDO token." into the charter as the example. I expect that @nadalin would prefer that we call out ways in which we intend integration to happen between DIDs and WebAuthn.

Reply:
good suggestion

Reply:
The RWoT6 DID Auth paper has a [short section](https://github.com/WebOfTrustInfo/rwot6-santabarbara/blob/master/final-documents/did-auth.md#webauthn) on one way of doing DID+WebAuthn integration (there may be several different ways), and IIW26 had a session with a [few slides](https://drive.google.com/drive/folders/1LyYp_SZpqboIPfUa1lo9zKtNv9SIv-5I) on this topic.